apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/Actor/Types/ActorGridAreaType.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
+
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.Actor.Types;
+
+[ObjectType<ActorDto>]
+public static partial class ActorGridAreaType
+{
+    public static async Task<IEnumerable<GridAreaDto>> GetGridAreasAsync(
+        [Parent] ActorDto actor,
+        GridAreas.IGridAreaByIdDataLoader dataLoader)
```

Comment:
question: why is this `GridAreas` "prefix" needed?

Reply:
Just what vscode did to resolve it. Have added a using instead